### ISSUE TITLE
Any chance the program supports Arcelik as well?

### ISSUE THREAD

**Issue by unscel** [deiger/AirCon#37]
Hi there and thank you for the great work. I come across your imazing work once I searched for beko & homebridge keywords in google. I got excited as mqtt integration with the AC was something I was lookin for a long time. While I read through README
I realized "beko-eu" does not have a corresponding app and its link, is it still under development?- Regardless, there it is Beko Connect link for ios: "https://apps.apple.com/tr/app/beko-connect/id1234472623" 

Having said that, I also got excited as Beko and Arcelik were the same products manufactured by Arcelik under different brands for different regions and even uses the same AC control app except it's name. Beko Connect & Arcelik Connect ( https://apps.apple.com/tr/app/ar%C3%A7elik-connect/id1215277478?l ). I assume it is the company who supports Arcelik with their IOT solutions. (https://apps.apple.com/tr/app/cosa-smart-heating-and-cooling/id947887227)

Long story short, would it possible to make use of your code for Arcelik Smart AC 's as well, what would you require and how I can help you with that considering my lack of coding knowledge. If it is about taking apart the AC to find out what kind of module it uses - even I can try that. 

Regards and keep up the great work.

**Comment by deiger** [comment]
Hi,

Given that this app is only for iOS, I cannot tell if it relies on the same server and protocol. If it is, the Beko code should be valid.
If you have such an AC, you can try using the CLI with your credentials, and see if you get a response:
```
./query_cli.py --user foo@example.com --passwd my_pass --app beko-eu --config config.json
```
Opening the AC cover, and seeing what is written on the module would definitely be helpful. It is located on the right side of the AC.

**Comment by deiger** [comment]
Found the parallel app in Google Play, [here](https://play.google.com/store/apps/details?id=com.beko.connect).
Unfortunately it does not use the Ayla Networks API, so it's unlikely to fit your AC.

**Comment by unscel** [comment]
I guess so. CLI returns “unauthorized” for the server. Thanks for the quick response
